base/channels.jl

```diff
@@ -503,7 +503,7 @@ end
 Determines whether a [`Channel`](@ref) has a value stored in it.
 Returns immediately, does not block.
 
-For unbuffered channels returns `true` if there are tasks waiting on a [`put!`](@ref).
+For unbuffered channels, this returns `true` if there are tasks waiting on a [`put!`](@ref).
```

Comment:
style nit: the docstrings are actually supposed to be in the imperative tense, as if being read as a command for the implementation of the function rather than a description of it
```suggestion
For unbuffered channels, return `true` if there are tasks waiting on a [`put!`](@ref).
```

Reply:
I would love to have such details straight, but turns out I dont. Thanks for the correction!